features/networking/service.feature

```diff
@@ -153,3 +153,334 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-24668
+  Scenario: externalIP defined in service but no sepc.externalIP defined	
+    Given I have a project 
+    # Create a service with a externalIP in :proj1
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/externalip_service1.json | 
+    Then the step should fail
+    Then the output should contain "externalIPs have been disabled"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-24669
+  @admin
+  @destructive
+  Scenario: externalIP defined in service with set ExternalIP in allowedCIDRs	 
+    # Create additional network through CNO
+    # Get node_1's host IP and save to clipboard
+    Given environment has at least 2 nodes
+    And I store the nodes in the :nodes clipboard
+    Given I use the "<%= cb.nodes[0].name %>" node
+    And evaluation of `host_subnet(cb.nodes[0].name).ip` is stored in the :hostip clipboard
```

Comment:
@weliang such steps like `host_subnet(cb.nodes[0].name).ip` won't work on OVN. I understand OVN not yet support external IP but will be in future.

Can you replace it with at all places ```https://github.com/anuragthehatter/verification-tests/blob/master/features/step_definitions/networking.rb#L881```

Reply:
@anuragthehatter Fixed to use above to get node IP address, testing passed. Thanks!